expo/gbe/views/edit_act_techinfo_view.py

```diff
@@ -31,10 +31,13 @@
     LightingInfoForm,
     CueInfoForm,
     VendorCueInfoForm,
-
-)
+    )
 from scheduler.models import Event as sEvent
 from gbetext import default_update_act_tech
+from expo.settings import (
+        DATETIME_FORMAT,
+        DAY_FORMAT,)
```

Comment:
This seems weird - it imports DATETIME and DAY but later on uses TIME
?
It works, but it makes me wonder how.